frontend/src/scenes/session-recordings/player/PlayerMeta.tsx

```diff
@@ -120,6 +130,29 @@ export function PlayerMeta(): JSX.Element {
         )
     }
 
+    const windowOptions: LemonSelectOption<string | null>[] = [
+        {
+            label: <IconWindow value={currentWindowIndex} className="text-muted-alt" />,
+            value: null,
+            labelInMenu: (
+                <div className="flex flex-row gap-2 space-between items-center">
+                    Follow the user: <IconWindow value={currentWindowIndex} className="text-muted-alt" />
```

Comment:
Wasn't sure of the language on these labels...

Could this just be "Auto"??

Reply:
I like follow the user!